src/rp2040/hardware_regs/include/hardware/platform_defs.h

```diff
@@ -68,9 +68,14 @@
 #elif defined(SYS_CLK_MHZ)
 #define SYS_CLK_HZ ((SYS_CLK_MHZ) * _u(1000000))
 #else
+// PICO_CONFIG: PICO_USE_FASTEST_SUPPORTED_CLOCK, Use the fastest officially supported clock by default, type=bool, default=0, group=hardware_base
```

Comment:
Given that this setting is used by code within `rp2_common`, would it make sense to also add this `PICO_CONFIG:` to one of the rp2350 header files?